app/controllers/blog_controller.rb

```diff
@@ -1,5 +1,11 @@
 # frozen_string_literal: true
 
 class BlogController < ApplicationController
-  def index; end
+  def index
+      @posts = Post.all
+  end
+
+  def show
+    @post = Post.find_by slug: params[:slug]
```

Comment:
Notice I'm finding by slug instead of id. This is because I want my urls to look like `en/blog/first-post` instead of `/en/blog/1`